stripe/src/main/java/com/stripe/android/paymentsheet/BaseAddCardFragment.kt

```diff
@@ -277,6 +280,33 @@ internal abstract class BaseAddCardFragment(
                 )
             }
         cardMultilineWidget.postalCodeErrorListener = null
+
+        billingAddressView.postalCodeViewListener =
+            object : BillingAddressView.PostalCodeViewListener {
+                override fun onLosingFocus(country: Country?, isPostalValid: Boolean) {
+                    if (isPostalValid) {
+                        billingErrors.text = null
+                        billingErrors.isVisible = false
+                    } else {
+                        if (country == null || country.code == "US") {
+                            billingErrors.text = getString(R.string.address_zip_invalid)
+                        } else {
+                            billingErrors.text = getString(R.string.address_postal_code_invalid)
+                        }
+                        billingErrors.isVisible = true
+                    }
```

Comment:
Another way to write this might be:
`
        billingErrors.text = if (isPostalValid) {
            null
        } else {
            if (country == null || country.code == "US") {
                getString(R.string.address_zip_invalid)
            } else {
                getString(R.string.address_postal_code_invalid)
            }
        }
        billingErrors.isVisible = !billingErrors.text.isNullOrEmpty()`

Reply:
nice kotlin trick 👍